src/main/java/com/thed/zephyr/jenkins/reporter/ZfjReporter.java

```diff
@@ -37,9 +43,15 @@
 
 import static com.thed.zephyr.jenkins.reporter.ZfjConstants.*;
 
-public class ZfjReporter extends Notifier {
+public class ZfjReporter extends Notifier implements SimpleBuildStep{
+
+    private static final String ENSURE_PUBLISH_J_UNIT_TEST_RESULT_REPORT = "Please ensure \"Publish JUnit test result report is added\" as a post build action";
+
+    private static final String ERROR_PARSING_SUREFIRE_REPORTS = "Error parsing surefire reports.";
+
+    private static final String VERIFY_THE_JOB_CONFIGURATION = "Cannot Proceed. Please verify the job configuration";
```

Comment:
Why did you add this constants? It's weird to use constants for log messages, if the same message is used repeatedly you can just add a log method (i.e. `logSurefireError()`, `logJobConfigurationError()`, etc)